session-name-1/activities/activity-folder-1/sol/solution.py

```diff
@@ -3,9 +3,17 @@
 
 PORT = 8001
 
+if len(sys.argv) == 1:
+    print("Please provide at least one argument, \"local\" or \"remote\" depending on where you want to run the solution.")
+    print("To run the solution for a different server, you can put sys.argv[1] = IP, sys.argv[2] = PORT.")
+    exit(1)
```

Comment:
Also I was wondering: why do we use this separation between the IP and port variables and not employ a single variable called `URL`?

Reply:
I think it's better for the flexibility and readability of the code.

Reply:
Agree with @gabrielmocanu